docs/integration/DataTypingAndSchemasHandling.md

```diff
@@ -0,0 +1,246 @@
+
+#### [Logical types](https://avro.apache.org/docs/1.11.0/spec.html#Logical+Types)
+
+Conversion at source to the specific type means that behind the scene Nussknacker
+converts [primitive type](https://avro.apache.org/docs/1.11.0/spec.html#schema_primitive)
+to [logical type](https://avro.apache.org/docs/1.11.0/spec.html#Logical+Types) - java objects, consequently, the
+end-user has access to methods of these objects.
+
+| Avro Schema                              | Java type  | Sample               | Comment                                                       |
+|------------------------------------------|------------|----------------------|---------------------------------------------------------------|
+| decimal (bytes or fixed)                 | BigDecimal |                      |                                                               |
+| uuid (string)                            | UUID       |                      |                                                               |
+| date (int)                               | LocalDate  | 2021-05-17           | Timezone is not stored.                                       |
+| time - millisecond precision (int)       | LocalTime  | 07:34:00.12345       | Timezone is not stored.                                       |
+| time - microsecond precision (long)      | LocalTime  | 07:34:00.12345       | Timezone is not stored.                                       |
+| timestamp - millisecond precision (long) | Instant    | 2021-05-17T05:34:00Z | Timestamp (millis since 1970-01-01) in human readable format. |
+| timestamp - microsecond precision (long) | Instant    | 2021-05-17T05:34:00Z | Timestamp (millis since 1970-01-01) in human readable format. |
+
+[//]: # (Missing support for: Local timestamp millisecond precision, Local timestamp microsecond precision, Duration)
+
+#### [Complex types](https://avro.apache.org/docs/1.11.0/spec.html#schema_complex)
+
+| Avro Schema | Java type                                      | Comment                                                     |
+|-------------|------------------------------------------------|-------------------------------------------------------------|
+| array       | List[_]                                        |                                                             |
+| map         | Map[String, _]                                 | Key - value map, where key is always represented by String. |
+| record      | org.apache.avro.generic.GenericRecord          |                                                             |
+| enums       | org.apache.avro.generic.GenericData.EnumSymbol |                                                             |
+| fixed       | org.apache.avro.generic.GenericData.Fixed      |                                                             |
```

Comment:
Same as above. It's irrelevant what is the "java type". What is relevant is how it's handled by NU Typing system.